### ISSUE TITLE
build-ibmcloud-win2012r2-x64-2: The paging file is too small for this operation to complete.  

### ISSUE THREAD

**Issue by andrew-m-leonard** [AdoptOpenJDK/openjdk-infrastructure#2154]
Node: https://ci.adoptopenjdk.net/computer/build-ibmcloud-win2012r2-x64-2/
https://ci.adoptopenjdk.net/job/Test_openjdk11_j9_sanity.openjdk_x86-64_windows/481/consoleFull
```
09:41:06  ===============================================
09:41:06  java/lang/ProcessHandle/TreeTest.java
09:41:06  Total tests run: 5, Failures: 3, Skips: 0
09:41:06  ===============================================
09:41:06  
09:41:06  STDERR:
09:41:06  JVMJ9VM011W Unable to load j9jit29: The paging file is too small for this operation to complete. 
09:41:06  JVMJ9VM011W Unable to load j9gc29: The paging file is too small for this operation to complete. 
09:41:06  JVMJ9VM011W Unable to load j9vrb29: The paging file is too small for this operation to complete. 
09:41:06  JVMJ9VM011W Unable to load j9jvmti29: The paging file is too small for this operation to complete. 
09:41:06  JVMJ9VM011W Unable to load jclse29: The paging file is too small for this operation to complete. 
09:41:06  Error: Could not create the Java Virtual Machine.
09:41:06  Error: A fatal exception has occurred. Program will exit.
09:41:06  Unhandled exception
09:41:06  Type=Segmentation error vmState=0x00020011
09:41:06  Windows_ExceptionCode=c0000005 J9Generic_Signal=00000004 ExceptionAddress=00007FFDAC9AA423 ContextFlags=0010005f
09:41:06  Handler1=00007FFDAD4FE560 Handler2=00007FFDB53AAAF0 InaccessibleReadAddress=0000000000000030
09:41:06  Unhandled exception
09:41:06  Type=Segmentation error vmState=0x00020011
```

**Comment by andrew-m-leonard** [comment]
C: only has 7Gb free, possibly not enough disk space if page file is there

**Comment by lumpfish** [comment]
Similar failures seen on test-ibmcloud-win2012r2-x64-1 - https://ci.adoptopenjdk.net/job/Test_openjdk16_j9_sanity.openjdk_x86-64_windows/63/consoleFull

**Comment by lumpfish** [comment]
This might help resolve: https://github.com/eclipse/openj9/issues/10650

**Comment by Willsparker** [comment]
As per the OpenJ9 link, I've changed it from the system automatically setting the page file size, to a custom size, initial size is 3072MB and maximum size is 8192MB - Though this is still on the C:\ drive.

For good measure, I've cleared the machine a bit, by removing the old versions of OpenSSL(I kept `OpenSSL1.1.1j` and `OpenSSL1.1.1k`). I also removed a stray `workspace` folder from the `C:/` drive, that hadn't been used since 2020 (besides, we use the `E:/` drive for builds on that machine, so I assume it was just a leftover test of the system). That machine has `11.5GB` free now, so should be able to accommodate the max page file size.
